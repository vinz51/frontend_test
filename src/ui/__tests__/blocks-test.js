import renderer from "react-test-renderer";
import React from "react";
import MainContainer from "../blocks";

describe("MainContainer", () => {
  describe("Snapshots", () => {
    it("renders", () => {
      const myTreeComponent = renderer.create(<MainContainer />).toJSON();

      expect(myTreeComponent).toMatchSnapshot();
    });
  });
});
