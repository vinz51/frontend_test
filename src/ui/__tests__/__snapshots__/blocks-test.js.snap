// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContainer Snapshots renders 1`] = `
<div
  className="sc-AxjAm kIeaYs"
/>
`;
